examples/index.html

```diff
@@ -23,9 +23,11 @@ <h1>Demos</h1>
 					<li><a href="../elements/pfe-icon-panel/demo">pfe-icon-panel</a></li>
 					<li><a href="../elements/pfe-layouts/demo">pfe-layouts</a></li>
 					<li><a href="../elements/pfe-markdown/demo">pfe-markdown</a></li>
+					<li><a href="../elements/pfe-modal/demo">pfe-modal</a></li>
 					<li><a href="../elements/pfe-number/demo">pfe-number</a></li>
 					<li><a href="../elements/pfe-page-status/demo">pfe-page-status</a></li>
 					<li><a href="../elements/pfe-progress-indicator/demo">pfe-progress-indicator</a></li>
+					<li><a href="../elements/pfe-styles/demo">pfe-styles</a></li>
```

Comment:
I think the pfe-styles item can be removed

Reply:
ahh, pesky index file. Removed.

Reply:
Removed. Thanks.